examples/minimal-mdns/client.cpp

```diff
@@ -163,6 +169,8 @@ OptionSet cmdLineOptions = { HandleOptions, cmdLineOptionsDef, "PROGRAM OPTIONS"
                              "        How long to wait for replies\n"
                              "  --multicast-reply\n"
                              "        Do not request unicast replies\n"
+                             "  --trace-to <dest>\n"
+                             "        trace to the given destination.\n"
```

Comment:
How does one figure out what destinations are possible/supported?

Reply:
Added a string constant to help.

Will need to see how this evolves, as I imagine we may want to support logging to files and then the format will become somehow longer - I was thinging of things like `--trace_to log,perfetto,file:/path/to/file` as a readable thing (but we do not yet have that ... so problem for future us)

I imagine I would want some basic target help string here and then have a better README somewhere explaining things.